tailwind.config.js

```diff
@@ -4,9 +4,12 @@ module.exports = {
   theme: {
```

Comment:
A good idea moving forward to avoid having to dig through everywhere in the code if we want to change any colors would be to name these something like backgroundColor: colorname, foregroundColor: colorname, textColor: colorname, primaryButtonColor: colorname etc. That way if we ever want to change colors for text or buttons or something like that we can just change it here and all the components will get the new color.

Reply:
Yeah that's a good point. I'll try to slowly incorporate that